### DISCUSSION TITLE
many playerbot leg here

### DISCUSSION THREAD

**Discussion by xengine-qyt** [mod-playerbots/mod-playerbots#252] [Q&A]
hello everyone.,the issuse about:https://github.com/azerothcore/azerothcore-wotlk/discussions/18977

Upgrading the machine configuration is useless
No more than 100 people per map

any idea?

**Comment by Dreathean** [comment]
I don't know what the exact issue is, but honestly is 5000 playerbots not enough? Especially when you can keep the level ranges near your level so you see the majority of them.
